Cargo.toml

```diff
@@ -3,6 +3,10 @@ name = "csa"
 version = "0.1.0"
 authors = ["Tristram Gräbener <tristramg@gmail.com>"]
 
+[[bin]]
+name = "bench"
+path = "src/benchmark.rs"
```

Comment:
you can just put benchmark.rs in src/bin, and you'll have your bin without touching your Cargo.toml.

Reply:
Oh nice indeed!